src/app/components/PortraitVideoCarousel/PortraitCarouselNavigation/index.test.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { act, RefObject } from 'react';
+import {
+  screen,
+  render,
+  fireEvent,
+} from '#app/components/react-testing-library-with-providers';
+import NavigationButtons from '.';
```

Comment:
Should we give this the same name as the component?
```suggestion
import PortraitCarouselNavigation from '.';
```